x/staking/types/historical_info_test.go

```diff
@@ -1,10 +1,11 @@
-package types
+package types_test
```

Comment:
Why changing this? It is not relevant to this PR.

Reply:
I added a teststaking import, and it created import cycles. This solves the cycles.

Reply:
OK, thanks.